relnotes

```diff
@@ -325,7 +327,7 @@ generate_notes () {
                    "Specify on the command-line. Exiting..."
   fi
 
-  range="$start_tag..$release_tag"
+  range="$start_tag..$upstream_remote_name/$release_tag"
```

Comment:
Why is this necessary?

Reply:
TBH, I'm not so sure about this. I saw @wojtek-t used this as a workaround in https://github.com/kubernetes/release/issues/380
```
-  range="$start_tag..${release_tag:-$branch_head}"
+  range="$start_tag..upstream/${release_tag:-$branch_head}"
```